packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FlutterWebView.m

```diff
@@ -417,70 +406,72 @@ - (void)updateAutoMediaPlaybackPolicy:(NSNumber*)policy
   }
 }
 
-- (bool)loadRequest:(NSDictionary<NSString*, id>*)request {
+- (bool)loadRequest:(WKWebView *)webView request:(NSDictionary<NSString *, id> *)request {
```

Comment:
When updating the `(bool)loadUrl:(WKWebView *)webView url:(NSString *)url withHeaders:(NSDictionary<NSString *, NSString *> *)headers` method to be static, this method could also be marked static.